src/foam/u2/view/RichChoiceView.js

```diff
@@ -286,7 +307,7 @@ foam.CLASS({
               var index = 0;
               return this.E().forEach(sections, function(section) {
                 this
-                  .start().hide(!! section.hideIfEmpty && resp[index].value <= 0)
+                  .start().hide(!! section.hideIfEmpty && resp[index].value <= 0 || ! section.heading)
```

Comment:
Why do you want to hide sections that don't have headings?

Reply:
Oh, I see, you're just hiding the heading itself, not the whole section.

Reply:
It's not hiding the section, it's hiding the header is none is passed in. Also did this so the styles applied to self.myClass('heading') aren't applied into the view.